api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -40,7 +40,7 @@
   }
 
   @Override
-  @AuthorityRequired({Authority.MANAGE_GROUP})
+  @AuthorityRequired({Authority.REVIEW_ID_VERIFICATION})
```

Comment:
I'm not quite sure why this change was made. It seems like disabling user should maybe be a distinct authority altogether? Maybe we should leave as-is and create a separate follow-up ticket?

Also: as I went looking for some clearer description of what each of these authorities allows, I could't find one. If you're willing to clean things up a bit while you're here, having a one-line sentence describing each authority level in o.p.w.m.Authority.java seems like a reasonably canonical place to document these enums.

(Also also... I'm noticing an issue with the disableUser method itself – the groupName string isn't actually needed anymore, and we should be logging the admin user action with the true / false disabled value rather than the access level. I think this wasn't fully cleaned up when we refactored the access levels. I'm happy to create a small follow-up PR for that if you'd rather not get involved :P)

Reply:
I made this because I think we are moving more towards a place where we have user admins, and making this a distinct authority adds more overhead where I don't think it is necessary after our dataAccessLevel State Machine refactor.

The Authority.java file is autoGenerated, so we can't put a comment there. I can put a comment in the swagger though.

Agreed that the groupName isn't needed for disableUser. I think it would be valuable to pull it out, but didn't want to tackle that within the scope of this PR.

Reply:
Gotcha. So the idea would be that REVIEW_ID_VERIFICATION --> USER_ADMIN in an ideal world? I wonder how easily we can rename these enums... are they stored in the database as strings or integer values?

Maybe the safest alternative would be to create a brand-new authority w/ the right name, and migrate folks over as needed. More work though :\

Sorry I missed that the file is auto-gen. Swagger makes sense.

Reply:
I'll check with CH, if I recall correctly I think he did some work around enforcing/making sure that the enums don't change out from under us/are deterministic based on ordering.

Reply:
Follow-up tickets here:
https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&view=planning&selectedIssue=RW-2130&epics=visible
https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&view=planning&selectedIssue=RW-2131&epics=visible